source/includes/_endpoint-batch.md

```diff
@@ -62,6 +62,65 @@ The variables IDs must match those of the target dataset since their types will
 }
 ```
 
+
+#### Checking if an append will cause problems
+
+`/datasets/{id}/batches/compare/`
+
+
+An append cannot proceed if there are any situations on the datasets involved
+that will cause ambiguous situations. If such datasets were to be appended
+the server will return a 409 response.
+
+It is possible to verify for this conditions before trying the append using
+the batches compare endpoint.
+
+```
+GET /datasets/4bc6af/batches/compare/?dataset=http://beta.crunch.io/api/datasets/3e2cfb/
+```
+
+The response will contain a conflicts key that can contain either `current`, 
+`incoming` or `union` depending of the type and location of the problem. Always
+a 200 response with empty or any conflicts.
+
+ * `current` refers to issues find on the dataset where new data would be added
+ * `incoming` has issues on the far dataset that contains the new data to add
+ * `union` expresses problems on the combined variables(metadata) of the final dataset after append.`
+
+```json
+
+{
+    "union": {...},
```

Comment:
Still need an example here. The explanation below suggests that I'll see something like

```
{
    "union": {
        "Subvariables in different arrays per dataset": ["id1", "id2"]
    },
    "current": {
        "Variables missing alias": ["id1"]
    }
}
```

or similar. Really? Also, that seems inadequate for "union" because the ids are different in each dataset.